src/shared/pager.c

```diff
@@ -161,8 +170,12 @@ void pager_close(void) {
                 return;
 
         /* Inform pager that we are done */
-        safe_fclose(stdout);
-        safe_fclose(stderr);
+        fflush(stdout);
+        dup2(stored_stdout, STDOUT_FILENO);
```

Comment:
it would make sense to cast this explicitly to `(void)`, to make clear that we ignore the return value here on purpose. (this has the benefit that static checkers won't complain on the ignored ret val here)